packages/perspective-viewer-d3fc/src/js/legend/legend.js

```diff
@@ -78,7 +85,12 @@ function legendComponent(scrollLegend, scaleModifier) {
                 .style("z-index", "2")
                 .call(scrollLegend);
 
+            resizable.addCallbackToResize(function(direction) {
```

Comment:
`addCallbackToResize` should probably be called something like `onResize`?

If we wanted it to look like a d3 event, then it would be mode like `on("resize", direction => {/*...*/});`, but I'm not sure that's necessary here.

Reply:
Changed to follow that paradigm. I agree that it's more consistent with d3, and also more easily extensible this way.